### PR TITLE
fix problem of useStoryblokState with useEffect function dependencies

### PR DESCRIPTION
**Pull request by serGlazkov**
fix of the problem https://github.com/storyblok/storyblok-react/issues/876

### CODE REVIEW

File: lib/common/client.ts [hunk 1]
```diff
@@ -5,28 +5,23 @@ import { registerStoryblokBridge } from "@storyblok/js";
...
   initialStory = null,
   bridgeOptions = {},
-
 ) => {
-  let [story, setStory] = useState(initialStory);
+  const [story, setStory] = useState(initialStory);
 
-  const isBridgeEnable =
-    typeof window !== "undefined" &&
+  const storyId = (initialStory as any).internalId || initialStory.id;
```
Comment (whlufisc) [inline lib/common/client.ts:11]:
```suggestion
  const storyId = initialStory ? (initialStory as any).internalId || initialStory.id : 0;
```
As `initialStory` can be null

Comment (yannic-wtfoxtrot) [reply lib/common/client.ts:11]: We could also do something like this:
```suggestion
  const storyId = (initialStory as any)?.internalId ?? initialStory?.id;
```
I'm not 100% sure about the `0` else case though.

Comment (serGlazkov) [reply lib/common/client.ts]: > Just to be save I would suggest the following change:
> 
> As `initialStory` can be null

fixed

**Comment by arorachakit** [comment]
Hey @serGlazkov ! 
Thank you for the pull request. 
We just had an internal review of the PR, and it seems good to go! 
I am merging it right now :) 

The SDK should work properly after this merge.

**Pull request merged**